README.md

```diff
@@ -79,6 +79,26 @@ Save and exit the file(:wq in vim). Use the following command to run your first
 ```
 Congratulations!! You ran your first gwion program.
 
+## Making Sound
+Gwion relies on plugins for most of its language features, including all those that make sound. Plugins are located in the subdirectories of
+`plug`. To get some sounds going under linux using jack sound server, you can build the plugins `Jack`, `Soundpipe`, and `Modules`.
+
+Starting from the Gwion base directory, to build `Jack`:
+```
+cd plug/Jack
+make
+```
+This will give you a shared object file, `Jack.so`. The default place Gwion will look for plugins is in a subdir of your home directory 
+named `.gwplug`. So create that directory and move `Jack.so` there:
+```
+mkdir ~/.gwplug
+mv Jack.so ~/.gwplug
+```
+Repeat for the other plugins mentioned. The `Soundpipe` plugin requires the soundpipe library, which we hope to have build seamlessly for you when you build the `Soundpipe` module, but we're not quite there yet. Please ask for help if this isn't working.
+
+When all those plugin `.so` files are in your `~/.gwplug` directory, you should be able to run a Gwion program that makes sound! In `plug/Modules` there's a `test.gw` program which plays a sine wave for 5 seconds. If the `gwion` you built is still in the base dir of your cloned repo, from the
+`plug/Modules` subdirectory you should be able to run `../../gwion -d jack test.gw` and hear some sound!
+
```

Comment:
Maybe add something along those lines
```md
> Currently those instructions are Linux-centric, we are working to improve that.
```

Reply:
Added to the top of the "making sound" section.